tokens/design-tokens.tokens.json

```diff
@@ -4165,58 +4165,22 @@
         }
       },
       "04": {
-        "0": {
-          "type": "custom-gradient",
-          "value": {
-            "gradientType": "linear",
-            "rotation": 134.9999995731132,
-            "stops": [
-              {
-                "position": 0.02,
-                "color": "#e32d00ff"
-              },
-              {
-                "position": 0.79,
-                "color": "#e5007bff"
-              }
-            ]
-          }
-        },
-        "1": {
-          "type": "custom-gradient",
-          "value": {
-            "gradientType": "linear",
-            "rotation": 180,
-            "stops": [
-              {
-                "position": 0.07,
-                "color": "#f7c1004d"
-              },
-              {
-                "position": 0.61,
-                "color": "#f7c10000"
-              }
-            ]
-          }
-        },
-        "2": {
-          "type": "custom-gradient",
-          "value": {
-            "gradientType": "linear",
-            "rotation": 180,
-            "stops": [
-              {
-                "position": 0.57,
-                "color": "#871cd900"
-              },
-              {
-                "position": 1,
-                "color": "#871cd940"
-              }
-            ]
-          }
-        },
         "description": "CTA bg - Active tabs and tooltips",
+        "type": "custom-gradient",
+        "value": {
+          "gradientType": "linear",
+          "rotation": 134.9999995731132,
+          "stops": [
+            {
+              "position": 0.02,
+              "color": "#e32d00ff"
+            },
+            {
+              "position": 0.79,
+              "color": "#e5007bff"
+            }
+          ]
```

Comment:
@aguscruiz, do you know the cause of this change?

Reply:
Uhm, not really, maybe the gradient color was changed by someone, but I don't remember doing it (it's possible I did it though hah)

Reply:
Should we go ahead and merge despite the unknown of this change?

Reply:
I think so, it must've been a cosmetic change